cmake/cmake_findExternalLibs.cmake

```diff
@@ -234,8 +234,6 @@ if (WITH_GUI)
     message(STATUS "Qt6Widgets not found!")
     message(FATAL_ERROR "To find a custom Qt installation use: cmake <..more options..> -DCMAKE_PREFIX_PATH='<path_to_parent_folder_of_lib_folder_withAllQt6Libs>' <src-dir>")
   ENDIF()
-  ## QuickWidgets is a runtime-only dependency that we need to copy and install when WebEngine is found.
-  find_package(Qt6 QUIET COMPONENTS ${OpenMS_GUI_QT_COMPONENTS_OPT} QuickWidgets)
```

Comment:
Aah no 😅 you still need to search for the optional components. Just not for the hard-coded QuickWidgets

Reply:
So COMPONENTS takes multiple arguments, the first of which here is ${OpenMS_GUI_QT_COMPONENTS_OPT} and the last is QuickWidgets?

So something like:
```find_package(Qt6 QUIET COMPONENTS ${OpenMS_GUI_QT_COMPONENTS_OPT})```
Would still look for the optional components, but not explicitly add QuickWidgets on to that list?

Reply:
Yes right!